spec/libsass-todo-issues/issue_947/expected.compact.css

```diff
@@ -0,0 +1,2 @@
+@keyframes test { 10% { color: red; } }
+
```

Comment:
I guess the trick is getting our of people's way as much as possible. Trying to manage newlines, indentation etc is barrier to people contributing. Especially when we can automatically fix styling ourselves with some simple tooling. I'm fine to just yolo it for now until we're at parity with Ruby Sass.